helpers/helpers_emba_prepare.sh

```diff
@@ -470,21 +471,35 @@ detect_root_dir_helper() {
   for R_PATH in "${ROOTx_PATH[@]}"; do
     if [[ -d "$R_PATH" ]]; then
       ROOT_PATH+=( "$R_PATH" )
-      if [[ -n "$MECHANISM" ]] && ! echo "$MECHANISM" | grep -q "file names"; then
-        MECHANISM="$MECHANISM / file names"
-      elif ! echo "$MECHANISM" | grep -q "binary interpreter"; then
-        MECHANISM="file names"
+       if [[ -z "$MECHANISM" ]]; then
+        MECHANISM="busybox"
+      elif [[ -n "$MECHANISM" ]] && ! echo "$MECHANISM" | grep -q "busybox"; then
+        MECHANISM="$MECHANISM / busybox"
+      fi
+      fi
```

Comment:
something happens here

Reply:
Sorry, was some copy and paste issue and I have set my editor to 2 spaces now